### DISCUSSION TITLE
[RedirectHttpsFilter] - Improve excludePaths

### DISCUSSION THREAD

**Discussion by Iilun** [playframework/playframework#12776] [Ideas]
Hello everyone,

I am trying to use the [RedirectHttpsFilter](https://www.playframework.com/documentation/2.9.x/RedirectHttpsFilter#Enabling-the-HTTPS-filter). I did not find in the documentation or source code a way to exclude a path that contains path attributes (`/foo/:name`). 

The current exclude path checks for exact matches. It could maybe be improved with wildcard patterns or route annotation (similar to `+nocsrf` for [CSRFFilter](https://www.playframework.com/documentation/2.9.x/ScalaCsrf#Applying-a-global-CSRF-filter)).

If this seems like a good idea, i could try a draft to implement it.

Thanks for your time

**Comment by mkurz** [comment]
> If this seems like a good idea, i could try a draft to implement it.

Yes, please provide a pull request.
IMHO this should be implement via a [`blackList`/`whiteList` approach](https://github.com/playframework/playframework/blob/main/web/play-filters-helpers/src/main/resources/reference.conf) that we already use for route modifiers like `+nocsrf` and `+nocsp`, etc.

**Reply by mkurz** [reply]
- fyi #12788 was merged

**Reply by Iilun** [reply]
Thanks !
